server/modules/cache/service.js

```diff
@@ -1,13 +1,26 @@
-import uuid from 'uuid/v4';
-import { TimeCache } from '@financial-times/n-map-cache-light';
+import MaskLogger from '@financial-times/n-mask-logger';
+const logger = new MaskLogger(['email', 'password']);
 
-const cache = new TimeCache();
+function btoa (str) {
```

Comment:
I know these are standard names (TIL), but it's much easier to read as `encode` and `decode`. It would clean up the file a little naming them as such as well.